style.min.css

```diff
@@ -1 +1 @@
amburger .hamburger-box{margin-top:-2px;width:22px}.is-size-desktop-medium .hamburger .hamburger-inner{margin-top:-1px}.is-size-desktop-medium .hamburger .hamburger-inner,.is-size-desktop-medium .hamburger .hamburger-inner::after,.is-size-desktop-medium .hamburger .hamburger-inner::before{width:22px;height:2px}.is-size-desktop-medium .hamburger .hamburger-inner::before{top:-7px}.is-size-desktop-medium .hamburger .hamburger-inner::after{bottom:-7px}.is-size-desktop-large .hamburger .hamburger-box{margin-top:-2px;width:31px}.is-size-desktop-large .hamburger .hamburger-inner{margin-top:-1.5px}.is-size-desktop-large .hamburger .hamburger-inner,.is-size-desktop-large .hamburger .hamburger-inner::after,.is-size-desktop-large .hamburger .hamburger-inner::before{width:31px;height:3px}.is-size-desktop-large .hamburger .hamburger-inner::before{top:-9px}.is-size-desktop-large .hamburger .hamburger-inner::after{bottom:-9px}}@media screen and (max-width:959px){.is-size-tablet-small .hamburger .hamburger-box{margin-top:-2px;width:19px}.is-size-tablet-small .hamburger .hamburger-inner{margin-top:-1px}.is-size-tablet-small .hamburger .hamburger-inner,.is-size-tablet-small .hamburger .hamburger-inner::after,.is-size-tablet-small .hamburger .hamburger-inner::before{width:19px;height:2px}.is-size-tablet-small .hamburger .hamburger-inner::before{top:-6px}.is-size-tablet-small .hamburger .hamburger-inner::after{bottom:-6px}.is-size-tablet-medium .hamburger .hamburger-box{margin-top:-2px;width:22px}.is-size-tablet-medium .hamburger .hamburger-inner{margin-top:-1px}.is-size-tablet-medium .hamburger .hamburger-inner,.is-size-tablet-medium .hamburger .hamburger-inner::after,.is-size-tablet-medium .hamburger .hamburger-inner::before{width:22px;height:2px}.is-size-tablet-medium .hamburger .hamburger-inner::before{top:-7px}.is-size-tablet-medium .hamburger .hamburger-inner::after{bottom:-7px}.is-size-tablet-large .hamburger .hamburger-box{margin-top:-2px;width:31px}.is-size-tablet-large .hamburger .hamburger-inner{margin-top:-1.5px}.is-size-tablet-large .hamburger .hamburger-inner,.is-size-tablet-large .hamburger .hamburger-inner::after,.is-size-tablet-large .hamburger .hamburger-inner::before{width:31px;height:3px}.is-size-tablet-large .hamburger .hamburger-inner::before{top:-9px}.is-size-tablet-large .hamburger .hamburger-inner::after{bottom:-9px}}@media screen and (max-width:576px){.is-size-mobile-small .hamburger .hamburger-box{margin-top:-2px;width:19px}.is-size-mobile-small .hamburger .hamburger-inner{margin-top:-1px}.is-size-mobile-small .hamburger .hamburger-inner,.is-size-mobile-small .hamburger .hamburger-inner::after,.is-size-mobile-small .hamburger .hamburger-inner::before{width:19px;height:2px}.is-size-mobile-small .hamburger .hamburger-inner::before{top:-6px}.is-size-mobile-small .hamburger .hamburger-inner::after{bottom:-6px}.is-size-mobile-medium .hamburger .hamburger-box{margin-top:-2px;width:22px}.is-size-mobile-medium .hamburger .hamburger-inner{margin-to
```

Comment:
does this file include any changes? I'm not seeing any `.scss` file committed, we can remove them from PR if so.

Reply:
Reverted them, don't know why they've changed.